cherrypy/_cperror.py

```diff
@@ -123,6 +123,9 @@ class Root:
 import six
 from six.moves import urllib
 
+from jaraco.classes.properties import classproperty
```

Comment:
I've been using a convention of creating four sections for imports: stdlib, compatibility (stdlib functionality but from third-party modules), third-party modules, and finally intrapackage imports. Perhaps my convention diverges slightly from the more common convention of putting all third-party imports (even six) in the same group.